editors/code/src/inlay_hints.ts

```diff
@@ -90,7 +92,14 @@ class HintsUpdater {
 
     private get allEditors(): vscode.TextEditor[] {
         return vscode.window.visibleTextEditors.filter(
-            editor => editor.document.languageId === 'rust',
+            editor => {
+                if (editor.document.languageId !== 'rust') {
```

Comment:
My slight preference would be for `editor => editor.document.languageId && editor.document.uri.scheme !== 'git' && editor.document.uri.scheme !== 'svn'`.